packages/compass/src/main/auto-update-manager.ts

```diff
@@ -579,9 +579,44 @@ class CompassAutoUpdateManager {
       ...options,
     };
 
+    // TODO: does it make sense to still support this? Should the menu item just
```

Comment:
With the noop autoupdater can we still at least tell the user that there is an update and let them download it? Does that actually work?